src/applications/disability-benefits/all-claims/pages/contactInformation.js

```diff
@@ -15,132 +14,17 @@ import {
   forwardingAddressDescription,
   ForwardingAddressViewField,
   phoneEmailViewField,
-  PrimaryAddressViewField,
 } from '../content/contactInformation';
 
-import {
-  isInFuture,
-  validateMilitaryCity,
-  validateMilitaryState,
-  validateZIP,
-} from '../validations';
-
-import { hasForwardingAddress, forwardingCountryIsUSA } from '../utils';
+import { isInFuture } from '../validations';
 
 import {
-  ADDRESS_TYPES,
-  MILITARY_CITIES,
-  MILITARY_STATE_LABELS,
-  MILITARY_STATE_VALUES,
-  STATE_LABELS,
-  STATE_VALUES,
-  USA,
-} from '../constants';
-
-/**
- *
- * @param {('addressCard.mailingAddress' | 'forwardingCard.forwardingAddress')} addressPath used for path lookups
- * @param {string} [title] Displayed as the card title in the card's header
- * @returns {object} UI schema for an address card's content
- */
-const addressUISchema = (addressType, title) => {
```

Comment:
Moved this and all its dependencies to [here](https://github.com/department-of-veterans-affairs/vets-website/pull/9588/files#diff-98f463a5c9f4478a5e22c14cb08dd953R324).